docs/en-US/component/dialog.md

```diff
@@ -175,6 +175,12 @@ When using `modal` = false, please make sure that `append-to-body` was set to **
 | open-auto-focus  | triggers after Dialog opens and content focused  | ^[Function]`() => void` |
 | close-auto-focus | triggers after Dialog closed and content focused | ^[Function]`() => void` |
 
+### Exposes
+
+| Name         | Description   | Type                    |
+| ------------ | ------------- | ----------------------- |
+| resetPostion ^(2.8.0) | reset postion | ^[Function]`() => void` |
```

Comment:
The version number is added here, which causes the width to change. It should be formatted to be consistent with the header.